File: client/web/src/types/theme.ts
export type Theme = "clothing" | "food" | "housing" | "transport";

export const THEME_TABS: Array<{ key: Theme; label: string }> = [
  { key: "clothing", label: "衣" },
  { key: "food", label: "食" },
  { key: "housing", label: "住" },
  { key: "transport", label: "行" },
];

File: client/ios/Packages/SimpleLivingCore/Sources/Core/Model/Theme.swift
import Foundation

public enum Theme: String, CaseIterable, Identifiable, Sendable {
    case clothing
    case food
    case housing
    case transport

    public var id: String { rawValue }

    public var title: String {
        switch self {
        case .clothing: return "衣"
        case .food: return "食"
        case .housing: return "住"
        case .transport: return "行"
        }
    }
}
